the-way-we-work.md

```diff
@@ -0,0 +1,664 @@
+- ensure Jira tickets are created for these objectives
+- select at least one of these Jira tickets to work on in each sprint
+- ensure this ticket(s) are included in the sprint backlog
+
+**SDMT Operational Planning**
+
+This work will be carried out in collaboration with SDMT.
+
+You will:
+
+- attend the Senior Developers Management Team (SDMT) meeting each week
+- chair SDMT on a rotating basis, preparing the agenda and writing meeting minutes
+- share important developments in your team or area of work
+- raise and discuss problems that require cross-team collaboration to solve
+- offer your assistance to help solve cross-team problems
+- help to maintain the SDMT Plan
+
+**Strategy Implementation and Lean Improvement Projects**
+
+This work will be carried out in collaboration with the leadership team.
+
+You will:
+
+- identify and agree priority improvement areas with the Dev & Test Manager
+- select one area per quarter to target for improvements as a personal objective
+- design lean improvement projects to deliver improvements across multiple products
+- lead, motivate and hold to account other developers to deliver the desired improvements
+
+**Vision and Strategy**
+
+This work will be carried out in collaboration with the leadership team.
+
+You will:
+
+- define and communicate a clear vision for NHSUK software development
+- collaborate with the leadership team to agree a strategy to deliver the vision
+
+
+### 3.3 Growing the Team
+
+**Code Reviews**
+
+This work will be carried out within a delivery team.
+
+You will:
+
+- agree a code review checklist with your team
+- invest enough time to provide a thorough review
+- review other people's code in before writing your own
+- use the code review as an opportunity to provide constructive advice and teaching
+
+**Mentoring and Pairing**
+
+This work will be carried out within a delivery team.
+
+You will:
+
+- actively seek out opportunities to develop the skills of other developers
+- pair with other developers whenever possible to complete feature tickets
+
+**Coaching**
+
+This work will be carried out on a one-to-one basis by arrangement with other developers.
+
+You will:
+
+- design programmes of coaching to improve developers' knowledge of agile software development
+- bprovide one-to-one coaching to support developers through their immediate and specific work issues
+- promote a coaching culture in NHSUK software development
+
+**Delegation**
+
+This work will be carried out in collaboration with the leadership team.
+
+You will:
+
+- identify opportunities to stretch and empower other members of the leadership team
+- delegate responsibility for the completion of significant work packages (e.g. lean improvement projects)
+- ensure resources are made available as agreed (e.g. developer time)
+- provide support and guidance where requested or required
+- champion the activities of other members of the leadership team (e.g. via all-hands presentations)
+
```

Comment:
Knowledge share and skill up tech team

Reply:
I think line 315 covers this with "identify opportunities to stretch and empower other members of the leadership team" but do come back to me if you mean something else